apps/Gruntfile.js

```diff
@@ -358,98 +359,89 @@ module.exports = function (grunt) {
-        'levels/dialogHelper': './src/code-studio/levels/dialogHelper.js',
-        'initApp/initApp': './src/code-studio/initApp/initApp.js'
-      },
-      commonFile: 'code-studio-common',
-      provides: ['marked', 'react', 'react-dom', 'radium']
-    },
+      entries: _.extend(
+        {},
+        appsEntries,
+        codeStudioEntries,
+        {
+          plc: './src/code-studio/plc/plc.js',
 
-    {
-      uniqueName: 'plc',
-      entries: {
-        plc: './src/code-studio/plc/plc.js'
-      },
-      provides: ['marked']
-    },
 
-    // Build embedVideo.js in its own step (skipping factor-bundle) so that
-    // we don't have to include the large code-studio-common file in the
-    // embedded video page, keeping it fairly lightweight.
-    // (I wonder how much more we could slim it down by removing jQuery!)
-    // @see embed.html.haml
-    {
-      uniqueName: 'embedVideo',
-      entries: {
-        embedVideo: './src/code-studio/embedVideo.js'
-      },
-      provides: ['jquery']
-    },
+          // Build embedVideo.js in its own step (skipping factor-bundle) so that
+          // we don't have to include the large code-studio-common file in the
+          // embedded video page, keeping it fairly lightweight.
+          // (I wonder how much more we could slim it down by removing jQuery!)
+          // @see embed.html.haml
+          embedVideo: './src/code-studio/embedVideo.js',
 
-    // embedBlocks.js is just React, the babel-polyfill, and a few other dependencies
-    // in a bundle to minimize the amound of stuff we need when loading blocks
-    // in an iframe.
-    {
-      uniqueName: 'embedBlocks',
-      entries: {
-        embedBlocks: './src/code-studio/embedBlocks.js'
-      },
-      provides: ['react', 'react-dom', 'radium']
-    },
+          // embedBlocks.js is just React, the babel-polyfill, and a few other dependencies
+          // in a bundle to minimize the amound of stuff we need when loading blocks
+          // in an iframe.
+          embedBlocks: './src/code-studio/embedBlocks.js',
 
-    {
-      uniqueName: 'makerlabDependencies',
-      entries: {
-        makerlab: './src/code-studio/makerlab/makerlabDependencies.js'
-      },
-      provides: ['johnny-five', 'playground-io', 'chrome-serialport']
-    },
+          makerlab: './src/code-studio/makerlab/makerlabDependencies.js',
 
-    {
-      uniqueName: 'pd',
-      entries: {
-        pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx'
-      }
-    },
+          pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx',
 
-    {
-      uniqueName: 'publicKeyCryptography',
-      entries: {
-        publicKeyCryptography: './src/publicKeyCryptography/main.js'
-      },
-      provides: ['react', 'react-dom', 'radium']
+          publicKeyCryptography: './src/publicKeyCryptography/main.js',
+        }
+      ),
+      externals: [
+        {
+          'jquery': 'var $',
```

Comment:
Something like that? There are rails templates have a script tag just for jquery. I haven't done a review of what code is relying on that script tag, but I imagine it's for whatever remaining inline javascript code we have laying around.